components/single_image_view/single_image_view.tsx

```diff
@@ -210,40 +208,50 @@ export default class SingleImageView extends React.PureComponent<Props, State> {
         }
 
         return (
-            <div
-                className={classNames('file-view--single', permalinkClass)}
-            >
-                <div
-                    className='file__image'
-                >
+            <div className={classNames('file-view--single', permalinkClass)}>
+                <div className='file__image'>
                     {fileHeader}
-                    {this.props.isEmbedVisible &&
-                    <div
-                        className={classNames('image-container', permalinkClass)}
-                        style={imageContainerStyle}
-                    >
+                    {this.props.isEmbedVisible && (
                         <div
-                            className={classNames('image-loaded', fadeInClass, svgClass)}
-                            style={styleIfSvgWithDimensions}
+                            className={classNames(
+                                'image-container',
+                                permalinkClass,
+                            )}
+                            style={imageContainerStyle}
                         >
-                            <div className={classNames(permalinkClass)}>
-                                <SizeAwareImage
-                                    onClick={this.handleImageClick}
-                                    className={classNames(minPreviewClass, permalinkClass)}
-                                    src={previewURL}
-                                    dimensions={this.state.dimensions}
-                                    fileInfo={this.props.fileInfo}
-                                    fileURL={fileURL}
-                                    onImageLoaded={this.imageLoaded}
-                                    showLoader={this.props.isEmbedVisible}
-                                    handleSmallImageContainer={true}
-                                    enablePublicLink={this.props.enablePublicLink}
-                                    getFilePublicLink={this.getFilePublicLink}
-                                />
+                            <div
+                                className={classNames(
+                                    'image-loaded',
+                                    fadeInClass,
+                                    svgClass,
+                                )}
+                                style={styleIfSvgWithDimensions}
+                            >
+                                <div className={classNames(permalinkClass)}>
```

Comment:
Are any of the changes in this file needed for this PR, or are they simply cosmetic? If they're just cosmetic, would you mind reverting them to avoid conflicting with other PRs that might be making changes to this area?

Reply:
It can be prettier. I have autoformat enabled on a lot of projects. I'll revert all of that 👍